src/commands/ListProtectedRoomsCommand.ts

```diff
@@ -15,18 +15,20 @@ limitations under the License.
 */
 
 import { Mjolnir } from "../Mjolnir";
-import { RichReply } from "matrix-bot-sdk";
+import { Permalinks, RichReply } from "matrix-bot-sdk";
 
 // !mjolnir rooms
 export async function execListProtectedRooms(roomId: string, event: any, mjolnir: Mjolnir) {
-    let html = `<b>Protected rooms (${Object.keys(mjolnir.protectedRooms).length}):</b><br/><ul>`;
-    let text = `Protected rooms (${Object.keys(mjolnir.protectedRooms).length}):\n`;
+    // FIXME: this isn't good.
```

Comment:
You'll have to be more specific.

Reply:
I wrote that because I thought it would be blindingly obvious why it was bad, though perhaps i have fixed what was obvious and not deleted the comment.

Reply:
Regardless of whether it's obvious or not when you write it, it may not remain obvious after the next refactoring, so it's a good idea to be specific :)